CHANGELOG.md

```diff
@@ -6,7 +6,14 @@
 - Thanks to completion of Issues [#42](issues/42) and [#45](issues/45), there is no longer a need to use the `openmatch-base` image when building components of Open Match.  Each stand alone appliation now is self-contained in its `Dockerfile` and `cloudbuild.yaml` files, and builds have been substantially simplified.  **Note**: The default `Dockerfile` and `cloudbuild.yaml` now tag their images with the version number, not `dev`, and the YAML files in the `install` directory now reflect this.
  - This paves the way for CI/CD in an upcoming version.  
  - This paves the way for public images in an upcoming version!
-
+- The code to read pools from your `MatchObject` `Properties` JSON blob when you pass it into the Backend API and make Open Match 'native' protobuf `PlayerPool` messages out of them for use with the MMLogic API is deprecated and will be removed in a future version. When that happens, if you want to use the MMLogic API, you will be required to populate the MatchObjects's `pools` field yourself.
+- The same with the code that makes `Rosters` messages out of your `Properties` JSON blob.  You'll have to populate the `Rosters` field yourself to make use of the MMLogic API in upcoming versions.
+- Running MMFs as kubernetes `jobs` is deprecated.  This pattern and the `MMForc` component will be removed in a future version.
+- The Backend API no longer prepends the timestamp as part of the key under which results are stored in Redis. This simplifies and shortens the keys a bit.  Existing evaluators will need to be updated to parse the new key format when they read from the `proposalq`.
+- Evaluator sample in golang got a pretty big overhaul.
```

Comment:
The future direction being considered is to not store the proposals in redis but just pass them in memory to the evaluator. The current change that moves the evaluator into a separate harness assuming it can read proposals out of the db - which may not work in future. 

Given that Open Match will trigger evaluation passing in in-memory proposals, we can either move back to k8s job for evaluator (which may not have scaling concerns) - or for consistency - if we use the harness approach - the evaluator may implement an interface over which Open Match will trigger evaluation. 

We can either leave the k8s mechanism for evaluator as is - or - create an issue to have the harness implement a method to trigger evaluation in 0.5.

Reply:
Yeah what was discussed at some point was to put an API in front on evaluator when we're ready to get proposals out of redis.  MMFs would then call the evaluator API to send proposals to it when they are finished.  Best to let Unity drive that in a future version based on their experiences, I suspect.